pgctl/cli.py

```diff
@@ -147,7 +149,7 @@ def svstat(*services):
     ]
 
 
-def exec_(argv):  # pragma: no cover
+def exec_(argv, env=os.environ):  # pragma: no cover, pylint:disable=dangerous-default-value
```

Comment:
env=None / if env is None: would be the usual idiom, rather than disabling the fairly valid lint rule.

p3